tmpl/cookie-consent-policy.php

```diff
@@ -5,12 +5,11 @@
  * @package Altis-Consent
  */
 
-$options     = get_option( 'cookie_consent_options' );
-$policy_page = $options['policy_page'];
+use Altis\Consent;
 ?>
 
 <div class="cookie-consent-policy">
-	<a href="<?php echo esc_url( get_permalink( (int) $policy_page ) ); ?>">
+	<a href="<?php echo esc_url( Consent\get_cookie_policy_url() ); ?>">
```

Comment:
Does it matter if this value might be empty?

Reply:
If it's empty, it's just an empty `<a>` tag (basically equivalent to a `<a href="#">` placeholder, and no error -- I think in practice that reloads the current page. Again, better than linking to a 404 and the same behavior that `get_privacy_policy_url` does.